content/blog/2021-09-21-collaborative-experiments.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Running Collaborative Experiments
+date: 2021-09-21
+description: |
+  Sharing experiments with teammates can help you build models more efficiently.
+descriptionLong: |
+  You can use DVC remotes to share experiments and their data across machines.
+picture: 2021-09-21/collaborative-experiments.png
+pictureComment: Running Collaborative Experiments
+author: milecia_mcgregor
+commentsUrl: https://discuss.dvc.org/t/tuning-hyperparameters-with-reproducible-experiments/821
+tags:
+  - MLOps
+  - DVC
+  - Git
+  - Experiments
+  - Collaboration
+---
+
+## Intro
+
+Sharing experiments to build machine learning model is important in order to
+create the best model as quickly as possible. You might have a new engineer join
+the team and you want to get their workspace ready quickly. Situations come up
+where you want to run experiments in differnt environments and you need to
+reproduce them.
+
+Setting up DVC remotes lets you share all of the data, code, and hyperparameters
+associated with each experiment so anyone can pick up where you left off in the
+training process. We'll go through an example of sharing an experiment with DVC
+remotes.
+
+## Getting the project
+
+To follow along, you can fork
+[this repo](https://github.com/iterative/dvc-learn-project) as one of your own
+GitHub repos. That way you'll have push and pull access when we start working
+with DVC. You do need to run the following command to be able to use the DVC
+CLI.
+
+```dvc
+dvc init
+```
+
+This project already has DVC files set up to run experiments, but if you want to
+follow along with a project you're currently working on, make sure to check out
+the steps to initialize a DVC pipeline in
+[the Getting Started doc](https://dvc.org/doc/start).
+
+## Setting up your DVC remotes
+
+When you want to share the progress you've made with training your model, that
+usually means you need to find a way to bundle the code, data, and
+hyperparameters. This could be a complicated process if you're working with GBs
+worth of data or you have a large number of hyperparameters.
+
+That's one of the uses for DVC and why we'll be working with remotes. To start
+with, make sure your GitHub remote is configured correctly. It should use the
+SSH version of the URL. This is so DVC can authenticate the pushes and pulls
+from GitHub it needs as part of experiment sharing.
+
+_We do have HTTPS support coming. You can follow our progress in
```

Comment:
The linked PR is not adding HTTPS support but rather about raising a more helpful exception message pointing to docs. 

@dberenbaum may we open a new issue regarding supporting HTTPS on exp commands (through prompt + memorization?)

Reply:
Yup, sorry, the plans have changed several times I think.

Reply:
@flippedcoder you could replace the current link with https://github.com/iterative/dvc/issues/6586 . Although the "HTTPS support coming" could be rephrased as it is not even prioritized yet.

Reply:
I think as @daavoo has sort of implied that we may just want to leave out a mention of HTTPS support since it's not prioritized right now. Let us know if you think that's a major pain point for the blog post.

Reply:
It's fine if we leave that reference out, so I just removed it. It doesn't affect the rest of the post.